sweep.yaml

```diff
@@ -0,0 +1,52 @@
+# Sweep AI turns bugs & feature requests into code changes (https://sweep.dev)
+# For details on our config file, check out our docs at https://docs.sweep.dev/usage/config
+
+# This setting contains a list of rules that Sweep will check for. If any of these rules are broken in a new commit, Sweep will create an pull request to fix the broken rule.
+rules:
+- "All docstrings and comments should be up to date."
+- "Do not use large chunks of commented out code."
+- "Use consistent indentation and spacing."
+- "Avoid using inline styles and instead use CSS classes."
+- "Do not use unnecessary imports."
+- "Use meaningful variable and function names."
+- "All commits should use the conventional commit format."
+- "All PR names should also use the conventional commit format."
+
+# This is the branch that Sweep will develop from and make pull requests to. Most people use 'main' or 'master' but some users also use 'dev' or 'staging'.
+branch: 'main'
+
+# By default Sweep will read the logs and outputs from your existing Github Actions. To disable this, set this to false.
+gha_enabled: True
+
+# This is the description of your project. It will be used by sweep when creating PRs. You can tell Sweep what's unique about your project, what frameworks you use, or anything else you want.
+#
+# Example:
+#
+# description: sweepai/sweep is a python project. The main api endpoints are in sweepai/api.py. Write code that adheres to PEP8.
+description: ''
+
+# This sets whether to create pull requests as drafts. If this is set to True, then all pull requests will be created as drafts and GitHub Actions will not be triggered.
+draft: False
+
+# This is a list of directories that Sweep will not be able to edit.
+blocked_dirs: []
+
+# This is a list of documentation links that Sweep will use to help it understand your code. You can add links to documentation for any packages you use here.
+#
+# Example:
+#
+# docs:
+#   - PyGitHub: ["https://pygithub.readthedocs.io/en/latest/", "We use pygithub to interact with the GitHub API"]
+docs:
```

Comment:
change this section so it is valid YAML format. Use this as an example of the format I expect:

```yml
docs:
  - nextjs13:
      - https://nextjs.org/docs
      - We use nextjs 13 as the main technology in our stack. It serves as a
        React framework to efficiently build the user interface
  - tailwind:
      - https://tailwindcss.com/docs/installation
      - Tailwind is our primary styling framework. We use it to build components
        with the desired css styles.
  - cypress:
      - https://docs.cypress.io/guides/overview/why-cypress
      - Cypress is our e2e framework. We test every new flux a user can take
        with this library
  - flowbite-react:
      - https://www.flowbite-react.com/docs/getting-started/introduction
      - this library is a little components library we use to make our lives
        easier and not needing to create all components in our prototyping
        phase. The usage of this library should be reduced over time.
```